samples_project/Assets/ChangeMaterial.cs

```diff
@@ -0,0 +1,43 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Rendering;
+
+public class ChangeMaterial : MonoBehaviour
+{
+    // Start is called before the first frame update
+    void Start()
+    {
+        var pipeline = GraphicsSettings.defaultRenderPipeline;
+        SkinnedMeshRenderer mapMan = GetComponent<SkinnedMeshRenderer>();
+        Material mapManMat = null;
+
+        if (GraphicsSettings.renderPipelineAsset.name.Contains("HDRP"))
+        {
+            mapManMat = Resources.Load("MapmanForUnity/HDRP_Metallic_Standard/MapmanMaterialHDRP", typeof(Material)) as Material;
+        }
+        else if (GraphicsSettings.renderPipelineAsset.name.Contains("URP"))
```

Comment:
`GraphicsSettings.renderPipelineAsset` is deprecated, would maybe instead use [GraphicsSettings.defaultRenderPipeline](https://docs.unity3d.com/ScriptReference/Rendering.GraphicsSettings-defaultRenderPipeline.html)

https://docs.unity3d.com/ScriptReference/Rendering.GraphicsSettings-renderPipelineAsset.html